### ISSUE TITLE
extensions/v1beta1/Ingress is deprecated  not supported by Kubernetes v1.20+ clusters

### ISSUE THREAD

**Issue by haarchri** [safesoftware/helm-charts#65]
please take care of extensions deprecation in Kubernetes v1.20+ clusters

https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122

```
Diagnostics:
  kubernetes:extensions/v1beta1:Ingress (xxx/fmeserver-websocket-ingress):
    warning: extensions/v1beta1/Ingress is deprecated by networking.k8s.io/v1beta1/Ingress and not supported by Kubernetes v1.20+ clusters.
 
  kubernetes:extensions/v1beta1:Ingress (xxx/fmeserver-web-ingress):
    warning: extensions/v1beta1/Ingress is deprecated by networking.k8s.io/v1beta1/Ingress and not supported by Kubernetes v1.20+ clusters.
 
  kubernetes:extensions/v1beta1:Ingress (xxx/fmeserver-transact-ingress):
    warning: extensions/v1beta1/Ingress is deprecated by networking.k8s.io/v1beta1/Ingress and not supported by Kubernetes v1.20+ clusters.
```

**Comment by garnold54** [comment]
Thanks for the report! We have an internal issue tracking this that we should merge soon. I will keep this issue open until it is available on the public helm chart.

**Comment by garnold54** [comment]
Sorry for the delay in updating this issue. It should be fixed in our FME Server 2021.1 and later helm charts.
